File: 00-Code/Android/Media/AndroidMediaPlayer/app/src/main/java/com/ztiany/mediaplayer/android/MainActivity.java
package com.ztiany.mediaplayer.android;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.ztiany.mediaplayer.android.test.NormalActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void normalPlayer(View view) {
        startActivity(new Intent(this, NormalActivity.class));
    }

}

File: Android/00-Code/AAC-Sample/Paging-Sample/src/main/java/com/ztiany/acc/MainActivity.kt
package com.ztiany.acc

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    companion object {
        const val LIVE_DATA = "LiveData"
        const val RXJAVA = "RxJava"
        const val TYPE = "TYPE"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setupView()
    }

    private fun setupView() {
        btnLiveData.setOnClickListener {
            val intent = Intent(this, ContentActivity::class.java)
            intent.putExtra(TYPE, LIVE_DATA)
            startActivity(intent)
        }

        btnRxJava.setOnClickListener {
            val intent = Intent(this, ContentActivity::class.java)
            intent.putExtra(TYPE, RXJAVA)
            startActivity(intent)
        }

    }

}
